contents/handbook/company/team/paolo-damico.md

```diff
@@ -4,8 +4,26 @@ sidebar: Handbook
 showTitle: true
 ---
 
+## Q1 2022 Goals
+- **Product strategy.**
+  - 2x ambitious Product milestones (like Experimentation). Strategy >> Execution. Exciting/evangelism.
+  - Decentralize & scale customer feedback loop with Customer Success Team.
+  - Scale (make sure it works optimally) Design work with Eng Teams.
+  - Bring more industry context. TODO:
```

Comment:
Ideally you'd have a specific objective you'd want to solve with industry context... e.g. Understand the highest quality products in our industry and identify what we can do to achieve the same level of quality... or... Evaluate the tope enterprise collaboration tools and identify the essential capabilities to make collaboration successful in PostHog...